assignment1.txt

```diff
@@ -0,0 +1,10 @@
+1. The return zero is used because it tells the compiler that the code is done. Every time the code runs it checks for errors and the 0 tells the compiler that the program worked if it reached the 0.
+
+
+2. I don't see anything wrong with the code
```

Comment:
Okay, I just looked over it and I think maybe it needs %f or the float needs to be changed to a double, and it needs the #include <stdio.h> at the top. 

Thanks!

Best,
Emma


int main() {  
    float x = 41.5;  
    printf("The value of x is %d", x);  
}  

> On Jun 29, 2016, at 10:46 AM, Lesley Cordero <notifications@github.com> wrote:
> 
> In assignment1.txt <https://github.com/CPRO-Session1/Assignment1/pull/3#discussion_r68959229>:
> 
> > @@ -0,0 +1,10 @@
> > +1. The return zero is used because it tells the compiler that the code is done. Every time the code runs it checks for errors and the 0 tells the compiler that the program worked if it reached the 0.
> > +
> > +
> > +2. I don't see anything wrong with the code
> Unfortunately, there are a few errors! Look at the code again and comment back with any errors you see! If you still can't find anything wrong, see me and we can go over it together.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/CPRO-Session1/Assignment1/pull/3/files/22d3ad36c4a2393ac1602258298d6f3bb7df7654#r68959229>, or mute the thread <https://github.com/notifications/unsubscribe/AHiVTuMK0HXJ_7STzSU0Fl6Q_pJPEWUpks5qQoVTgaJpZM4JAiNe>.
>